assets/i18n/qqq.json

```diff
@@ -564,6 +564,7 @@
 	"I18N_LEARNER_DASHBOARD_VIEW_SUGGESTION": "The name of a button in the learner dashboard. On clicking it the contents of the suggestion are viewable.",
 	"I18N_LEARNT_TOPIC": "Text displayed on the learner dashboard that marks a topic as completed.",
 	"I18N_LEARN_TOPIC": "Text displayed on the learner dashboard that prompts user to start a topic.",
+	"I18N_LESSON_INFO_HEADER": "Text displayed in the left of exploration footer.",
```

Comment:
Please be more detailed about the "I18N_LESSON_INFO_HEADER", just take a look "I18N_LEARNER_DASHBOARD_VIEW_SUGGESTION" for reference, as "I18N_LESSON_INFO_HEADER" is also button text.

Reply:
Will change!